packages/eslint-plugin-pf-codemods/src/rules/v6/cardRemoveVariousProps/cardRemoveVariousPropsOutput.tsx

```diff
@@ -0,0 +1,3 @@
+import { Card } from "@patternfly/react-core";
+
+export const CardRemoveVariousPropsInput = () => <Card />;
```

Comment:
This output should include the whitespace left behind by the removed props:

```
  <Card
    
    
    
    
    
  />
```

Reply:
Whoops, blame prettier 😆